app/views/users/edit_profile.html.erb

```diff
@@ -1,17 +1,23 @@
 <h2><%= I18n.t('user.edit.edit_profile_settings_heading') %></h2>
 
-<%= form_with(model: @user, local: true) do |form| %>
-  <% if @user.errors.any? %>
-    <%= render 'layouts/flash_messages'%>
-  <% end %>
+<p>
+  <%= render 'avatar', user: @user, size: 160, classes: %i[rounded-circle centered-avatar] %>
```

Comment:
should `centered-avatar` be added to the `card.html` as well?

Reply:
It wasn't there before so I assumed that it is not necessary there. In my opinion it could be omitted completely as it does not look good on the profile page but this PR was more about making it work again, not making it pretty :sweat_smile: